### DISCUSSION TITLE
Custom Link for dcim/interfaces based on device.tenant

### DISCUSSION THREAD

**Discussion by vladimirs-git** [netbox-community/netbox#14193] [Q&A] [answered]
I need help creating a **Custom Link** for **dcim/interfaces** that selects a URL based on the device.tenant. Is this possible?

The following example is not functioning. It's just to demonstrate my objective.
`{% if object.device.tenant.name == "TENANT 1" %}
https://domain1.com
{% elif object.device.tenant.name == "TENANT 2" %}
https://domain2.com
{% endif %}`

**Comment by zmaster7** [comment, marked as answer]
I just tried a similar example to yours in my test environment (v3.6.4), and it is working as intended:

![image](https://github.com/netbox-community/netbox/assets/46574676/f24d724a-46bd-4ae6-b649-477499f31e05)

![image](https://github.com/netbox-community/netbox/assets/46574676/e4ca22d4-e360-4ddb-91c0-970a806305c5)

**Reply by zmaster7** [reply]
To dive a bit deeper, I went into nbshell and was able to extract the device's tenant name using the first Interface object as an example:

Interface.objects.first().device.tenant.name

**Comment by vladimirs-git** [comment]
Confirm, my example is working. Thank you for your help.
